code/modules/movespeed/modifiers/reagent.dm

```diff
@@ -5,7 +5,7 @@
 	multiplicative_slowdown = -0.55
 
 /datum/movespeed_modifier/reagent/ephedrine
-	multiplicative_slowdown = -0.35
+	multiplicative_slowdown = -0.5
```

Comment:
If you want to do purity, you'd set `variable = TRUE` here and remove this line, commenting that it gets set on add.